calico/felix/fsocket.py

```diff
@@ -213,6 +296,9 @@ def __init__(self, type, fields, endpoint_id=None):
         else:
             self.descr = type
 
+        #: Whether this is a keepalive
+        self.keepalive = keepalive
+
```

Comment:
Oops. An earlier version used this field, then I realised it was a bad idea and failed to tidy it up. So I have tidied it up now.